paasta_tools/cli/cmds/start_stop_restart.py

```diff
@@ -140,6 +140,19 @@ def print_chronos_message(desired_state):
         )
 
 
+def confirm_to_continue(cluster_service_instances, desired_state):
+    paasta_print(f'You are about to {desired_state} the following instances:')
+    i_count = 0
+    for cluster, services_instances in cluster_service_instances:
+        for service, instances in services_instances.items():
+            for instance in instances.keys():
+                paasta_print(f'cluster = {cluster}, instance = {service}')
```

Comment:
I think you mean {instance}, not {service} (look at the example output again)

Reply:
lol